usaspending_api/search/delta_models/award_search.py

```diff
@@ -299,6 +301,10 @@
   RL_STATE_POPULATION.latest_population AS recipient_location_state_population,
   RL_COUNTY_POPULATION.latest_population AS recipient_location_county_population,
   RL_DISTRICT_POPULATION.latest_population AS recipient_location_congressional_population,
+  CONCAT(
```

Comment:
CONCAT will return the value that does exist without the NULL column's value. It would look like you combined a string with letters with an empty string: `"hello" + "" = "hello"`